src/core/server/http/http_config.ts

```diff
@@ -70,6 +70,18 @@ const configSchema = schema.object(
     securityResponseHeaders: securityResponseHeadersSchema,
     customResponseHeaders: schema.recordOf(schema.string(), schema.any(), {
       defaultValue: {},
+      validate(value) {
+        const RESPONSE_HEADER_DENY_LIST = ['location'];
```

Comment:
I would extract the list from the `validate` method

Reply:
Done in [`0fec11b` (#98129)](https://github.com/elastic/kibana/pull/98129/commits/0fec11b78319c346272f4e874b60c6a06ebc9b71)